ur_description/urdf/ur10.urdf.xacro

```diff
@@ -299,6 +299,26 @@
 
     <xacro:ur10_arm_transmission prefix="${prefix}" />
     <xacro:ur10_arm_gazebo prefix="${prefix}" />
-  
+
+    <!-- ROS base_link to UR 'Base' Coordinates transform -->
+    <link name="${prefix}base"/>
+    <joint name="${prefix}base_link-base" type="fixed">
+      <!-- NOTE: this rotation is only needed as long as base_link itself is
+                 not corrected wrt the real robot (ie: rotated over 180
+                 degrees)
+      -->
```

Comment:
Yes, I think we do. First, it is a rationale for why there is a rotation between `base` and `base_link` in the first place, and secondly, I do believe we should fix-up `base_link`, but do so respecting bw-compatibility release practices (ie: tick-tock).